docs/hybrid_applications.md

```diff
@@ -6,6 +6,14 @@ You can use Datadog on the same application both from the React Native and the n
 
 RUM Events are reported as coming from the same application and the same source in Datadog RUM.
 
+Refer to the [`example apps`](https://github.com/DataDog/dd-sdk-reactnative-examples) to see a few implementations of Datadog with hybrid applications. The following scenarios are present for now:
+
+1. [`android-app-with-rn`](https://github.com/DataDog/dd-sdk-reactnative-examples/tree/main/android-app-with-rn) app (android app with RN screens)
+
+2. [`ios-app-with-rn`](https://github.com/DataDog/dd-sdk-reactnative-examples/tree/main/ios-app-with-rn) app  (iOS app with RN screens)
+
+3. [`rn-app-with-native`](https://github.com/DataDog/dd-sdk-reactnative-examples/tree/main/rn-app-with-native) app (RN app with native screens)
+
```

Comment:
I believe it would be better to have these examples in a dedicated section at the end of the page. We want users to follow the documentation first, then go to the examples if they need to see more, or we risk having them trying to replicate the example apps and missing things.

I would add a `#### Example` at the end of both `### React Native apps with native content: Initialize the React Native SDK` and `### Native apps with React Native screens: Initialize the SDK on the native side` with the links to the example apps.

Do you see any reason to keep them here?

Reply:
I was following the same logic for the others. I was putting the example on top. In this case I would support putting in an `#### example` header but in the other cases we indicated to refer to them when there's an error, why should this be any different? 
We can either choose to put them everywhere in `#### Examples` headers or just point them to it in case of errors. 
What do you think?